app/controllers/articles_controller.rb

```diff
@@ -27,7 +42,20 @@ def create
 
   private
 
+  def set_article
+    @article = Article.find(params[:id])
+  end
```

Comment:
I would change this method to use the `||=` pattern I showed you earlier. Then, I would change the name of the method to `article`, make it a helper method, and then use the method everywhere instead of `@article`.